acme_diags/derivations/acme.py

```diff
@@ -387,6 +387,8 @@ def cosp_histogram_standardize(cld):
         (('prsn',), lambda prsn: qflxconvert_units(rename(prsn))),
         (('PRECSC', 'PRECSL'), lambda precsc, precsl: precst(precsc, precsl))
     ]),
+    # Sea Surface Temperature: Degrees C
```

Comment:
@chengzhuzhang I added comments to some variables. If you feel strongly that these comments should not be here, I can take them out. That said, I think they really help aid understanding of what's going on in this file. (Rather than commenting on every variable in this file, I figure I can add comments as needed in this and future pull requests).

Reply:
Good idea.